packages/terra-clinical-detail-view/tests/wdio/DetailList/detail-list-spec.js

```diff
@@ -0,0 +1,15 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge');
+const selector = '#root';
+
+describe('Detail List', () => {
+  describe('Displays a default Detail List with a title and list', () => {
```

Comment:
In the old nightwatch, there wasn't a test for the `large-item-list` example as it was proving the list flex for large display items. Could we add a test step to capture this behavior?

Reply:
c981c852fc2c0da99711b31915c67acce79fb78c